src/platforms/f3/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## System vs BMP Bootloader
 
-For simpicity, the ST system bootloader is used. Thi saves additional 4 kB for the BMPfirmware.
+For simpicity, the ST system bootloader is used. This saves additional 4 kB for the BMPfirmware.
```

Comment:
There's a second correction here - `BMPfirmware` -> `Black Magic Firmware` or `BMP Firmware` - take your pick as to which.